recipes/kinbot/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# cmd: conda build . -c conda-forge
+
+{% set name = "kinbot" %}
+{% set version = "2.1.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: kinbot-2.1.post0.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kinbot-2.1.post0.tar.gz
```

Comment:
@ocefpaf : The version of KinBot is 2.1.0, however, an error in the build process for the PyPI package made necessary to make a post release version, as PyPI does not allow to upload files with the same name. Both 2.1.0 and 2.1.post0 are the same version of the code and only differ in the PyPI build. Because this has nothing to do with PyPI, I think the version should stay as 2.1.0.